syncthing-git/PKGBUILD

```diff
@@ -5,63 +5,66 @@
 # Previous Maintainer : Sauyon Lee <sauyonl@sauyon.com>
 # Contributor : Martin Wimpress <code@flexion.org>
 
-pkgname=syncthing-git
-pkgver=v1.5.0.rc.2.r18.g037934ec7
+_pkgname=syncthing
+pkgname=$_pkgname-git
+pkgver=v1.6.0.rc.4.r21.g668979605
 pkgrel=1
 pkgdesc="Open Source Continuous Replication / Cluster Synchronization Thing"
-url="http://syncthing.net/"
+url="https://syncthing.net/"
 license=('MPLv2')
 makedepends=('git' 'go' 'inetutils')
-conflicts=('syncthing')
-provides=('syncthing')
+conflicts=("$_pkgname")
+provides=("$_pkgname")
 arch=('i686' 'x86_64')
-source=("$pkgname::git+https://github.com/syncthing/syncthing.git")
+source=("git+https://github.com/syncthing/syncthing.git")
 sha256sums=('SKIP')
-install=${pkgname}.install
-_name=syncthing
+install=${_pkgname}.install
 
 prepare() {
-	cd "${srcdir}"
+	cd "$srcdir/$_pkgname"
+	mkdir -p "${srcdir}/../go"
+	export GOPATH="${srcdir}/../go"
+	export PATH="$PATH:$GOPATH/bin"
+	# fix for broken version
+	chmod u+w -R "$GOPATH"
+	go get -d -v ./...
 }
 
 pkgver() {
-	cd "${pkgname}"
+	cd "$srcdir/$_pkgname"
 	git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
```

Comment:
make sure this simple method works for the git repo you're quering. On the go-ipfs repo it will fail, since patch releases are sometimes in branches, leaving all releases based on the same commit on the master.

This will result in an older release being shown in the version tag, which is: technically right but very confusing.

Reply:
Until now it worked correctly and it also seems to me that they tag the master branch so it should be fine.